prow/cluster/prow_job.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
```

Comment:
nice catch. I created an[ issue upstream](https://github.com/kubernetes/test-infra/issues/3400), I assume the plan is to make the switch.